dbms/src/Common/StringUtils/StringUtils.h

```diff
@@ -155,6 +160,19 @@ inline bool isPunctuation(char c)
     return (c >= 0x21 && c <= 0x2F) || (c >= 0x3A && c <= 0x40) || (c >= 0x5B && c <= 0x60) || (c >= 0x7B && c <= 0x7E);
 }
 
+// return how many bytes the utf8 character occupies
+inline size_t getUTF8Length(uint8_t c)
```

Comment:
You can use `UTF8::seqLength` in dbms/src/Common/UTF8Helpers.h.

Reply:
> You can use `UTF8::seqLength` in dbms/src/Common/UTF8Helpers.h.

okk